apps/code-examples/src/app/code-examples/forms/input-box/basic/input-box-demo.component.html

```diff
@@ -8,156 +8,69 @@ <h2>New member form</h2>
       </sky-row>
       <sky-row>
         <sky-column [screenSmall]="12" [screenMedium]="6">
-          <sky-input-box class="sky-margin-stacked-lg">
-            <label class="sky-control-label" [for]="firstNameInput.id">
-              First name
-            </label>
-            <input
-              formControlName="firstName"
-              type="text"
-              spellcheck="false"
-              class="sky-form-control"
-              skyId
-              #firstNameInput="skyId"
-            />
+          <sky-input-box labelText="First name" stacked="true">
+            <input formControlName="firstName" type="text" spellcheck="false" />
           </sky-input-box>
         </sky-column>
         <sky-column [screenSmall]="12" [screenMedium]="6">
-          <sky-input-box class="sky-margin-stacked-lg">
-            <label class="sky-control-label" [for]="lastNameInput.id">
-              Last name
-            </label>
-            <input
-              formControlName="lastName"
-              type="text"
-              spellcheck="false"
-              class="sky-form-control"
-              [attr.aria-describedby]="lastNameError.id"
-              skyId
-              #lastNameInput="skyId"
-            />
-            <span class="sky-error-indicator" skyId #lastNameError="skyId">
-              <sky-status-indicator
-                *ngIf="
-                  (lastName.dirty || lastName.touched) &&
-                  lastName.errors &&
-                  lastName.errors['required']
-                "
-                indicatorType="danger"
-                descriptionType="error"
-              >
-                Last name is required.
-              </sky-status-indicator>
-            </span>
+          <sky-input-box labelText="Last name" stacked="true">
+            <input formControlName="lastName" type="text" spellcheck="false" />
           </sky-input-box>
         </sky-column>
       </sky-row>
       <sky-row>
         <sky-column [screenSmall]="12">
-          <sky-input-box class="sky-margin-stacked-lg">
-            <label class="sky-control-label" [for]="bioInput.id"> Bio </label>
-            <sky-help-inline
-              class="sky-control-help"
-              (actionClick)="onActionClick()"
-            ></sky-help-inline>
-            <textarea
-              formControlName="bio"
-              class="sky-form-control"
-              skyId
-              #bioInput="skyId"
-            ></textarea>
+          <sky-input-box
+            helpPopoverContent="A brief description of the member's background, such as hometown, school, hobbies, etc."
+            characterLimit="250"
```

Comment:
sooooooooooo I'm not seeing any character count or error appear when I type more then 250 characters into bio